debian/ubuntu-advantage-tools.postinst

```diff
@@ -421,14 +421,11 @@ case "$1" in
         fi
       fi
 
-      # We modify permissions for ubuntu-advantage.log because
-      # in a past version of UA, this log file was world readable.
-      # This isn't necessary for the timer/license-check log files,
-      # because they have always been only root-readable.
+      # /var/log/ubuntu-advantage.log needs to be world-readable
       if [ ! -f /var/log/ubuntu-advantage.log ]; then
           touch /var/log/ubuntu-advantage.log
       fi
-      chmod 0600 /var/log/ubuntu-advantage.log
+      chmod 0644 /var/log/ubuntu-advantage.log
```

Comment:
We probably also need something like this for the other log files as well:
```bash
if [ -f /var/log/ubuntu-advantage-timer.log ]; then
  chmod 0644 /var/log/ubuntu-advantage-timer.log
fi
if [ -f /var/log/ubuntu-advantage-license-check.log ]; then
  chmod 0644 /var/log/ubuntu-advantage-license-check.log
fi
```

Reply:
I have added the two, plus an extra one i.e. `/var/log/ubuntu-advantage-daemon.log`

Reply:
@orndorffgrant @renanrodrigo do we want to perform any additional testing before making the `ubuntu-advantage.log` world readable ?